### ISSUE TITLE
Referring to #113578

### ISSUE THREAD

**Issue by Krammig** [flutter/flutter#117848]
## Steps to Reproduce

1. ...
2. ...
3. ...

## Logs

<details>
<summary>Logs</summary>

```
```

```
```

</details>

**Comment by Krammig** [comment]
@Joebeazelman is absolutely correct. 

I have just followed the instructions here ; 

`https://docs.flutter.dev/get-started/install/macos`

and it does not work. 

Message ; 

Error: The Flutter directory is not a clone of the GitHub project.
       The flutter tool requires Git in order to operate properly;
       to install Flutter, see the instructions at:
       `https://flutter.dev/get-started`

Ridiculous.

I have previously installed fine on Android Studio - Windows version.

**Comment by darshankawar** [comment]
Thanks for the report @Krammig 
Although I haven't tried the installation steps yet to checkout myself, but can you confirm after cloning, the path to the flutter is updated in the `PATH` env variable and you are able to execute flutter run -v  ?

**Comment by Krammig** [comment]
Sure, 
Yes correct, the PATH environment variable has been updated and is pointing correctly. 
Running your requested flutter run -v results in the same message as stated above.

**Comment by Krammig** [comment]
I believe I have resolved the situation. 

a) 
Following the installation instructions for the downloading of the zip file (for Mac) definitely creates a problem. My feeling, there is a difference between the Mac and Windows zip process because the Windows process does not result in that message being displayed. So it seems to know, and is happy with the fact, that it was not a Git based install. 

b)
Following the Git install process on the Mac, only works, if you Do Not move the installed Flutter directory. 
There are hidden files in the directory after the Git clone which various Flutter commands seem to want to see. Why I am not sure, does not seem to a necessary really ?

If you move the Flutter directory to a different location, then these hidden files do not get moved this that Error message is displayed. 

As I said, installing via the Zip process gives the same result irrespective, and I assume for the same reason being the Hidden files are not being installed in the Zip process. 

IMHO there seems to be no reason why Flutter should be checking to see if those hidden files are present first. 

Cheers

**Comment by darshankawar** [comment]
Thanks for the update. I tried the macOS installation per the documentation, updated the path and then ran `flutter doctor` which ran properly and gave me the expected output, ie, I didn't get the reported error.

I installed the intel .zip version. Did you try the same or the apple silicon one ? Not sure if that will make any difference though..

Also, I am wondering if this need to be reported in flutter/website repo, since the instructions are coming from https://docs.flutter.dev/ and has a separate repo for tracking bugs.
